.github/workflows/deploy_develop.yml

```diff
@@ -46,6 +46,7 @@ jobs:
           ARAGON_ETH_SUBSCRIPTION_EVENT_DELAY: 5000
           ARAGON_FORTMATIC_API_KEY: ${{ secrets.FORMATIC_API_KEY }}
           ARAGON_PORTIS_DAPP_ID: ${{ secrets.PORTIS_DAPP_ID }}
+          ARAGON_WALLETCONNECT_URL: ${{ secrets.WALLETCONNECT_URL }}
```

Comment:
does this have to be a secret? or can it be hardcoded in the workflow?

Reply:
don't forget the same change in the `deploy_prod.yml` workflow file

Reply:
Change from 'unsupported' to 'not maintained' was a new tag I added some months ago, and Ramon suggested to edit it to 'not maintained', sounded clearer or smtin